src/styles/index.scss

```diff
@@ -1,3 +1,245 @@
+:root {
+  --light-blue: #00acdc;
+  --column-count: 4;
+}
+
+* {
+  box-sizing: border-box;
+}
+
 body {
   margin: 0;
+  font-family: Roboto, Arial, sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+  font-style: normal;
+}
+
+// #region header
+.header {
+  box-sizing: border-box;
+  padding-left: 50px;
+  padding-right: 50px;
```

Comment:
It's better to use shorthand properties when you can. This makes your code more concise and easier to read. You can replace padding-left and padding-right with the shorthand padding.